.docker/secrets/idp/idp-encryption.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Should this be checked in?

Reply:
These `secrets` files are copied from `jhuda-docker` repo and is also the same as is used in the local PASS dev environment. These keys are used only for local development environments